src/guiFormSpecMenu.cpp

```diff
@@ -746,7 +743,8 @@ void GUIFormSpecMenu::parseTable(parserData* data,std::string element)
 		spec.ftype = f_Table;
 
 		for (unsigned int i = 0; i < items.size(); ++i) {
-			items[i] = unescape_string(items[i]);
+			items[i] = unescape_string(wide_to_utf8(
+				remove_escapes(utf8_to_wide(items[i]))));
```

Comment:
Wide characters are not uniform across platforms. On some they are wider, on some they are smaller.